the_mines/rapidapi_test.py

```diff
@@ -0,0 +1,20 @@
+"""
+rapidapi requires credit card data even for free acount.
+free account gives you 50 free api calls per month and $0.25 for each call after that.
+NO THANKS!
+
+"""
```

Comment:
This is in the wiki now, we can probably take it out of here.

Reply:
I'm deleting the entire file